parm/archive/gefs_ocean.yaml.j2

```diff
@@ -0,0 +1,46 @@
+{% set cycle_HH = current_cycle | strftime("%H") %}
+{% set cycle_YMD = current_cycle | to_YMD %}
+{% set cycle_YMDH = current_cycle | to_YMDH %}
+{% set head = RUN + ".ocean.t" + cycle_HH + "z." %}
+
+gefs_ocean:
+    name: "GEFS_OCEAN"
+    target: "{{ ATARDIR }}/{{ cycle_YMDH }}/gefs_ocean.tar"
+    required:
+{% if CASE == "C48" %}
+    {% set res = '5p00' %}
+{% elif CASE == "C96" or "C192" %}
+    {% set res = '1p00' %}
+{% elif CASE == "C384" %}
+    {% set res = '0p25' %}
+{% elif CASE == "C768" %}
+    {% set res = '0p25' %}
+{% endif %}
```

Comment:
This should be based on `OCNRES` instead of `CASE`. Preferably in a compact way that just inserts the `p` three from the end (e.g. `500` ➡ `5p00`).